### PR TITLE
Product Variant Inactive

### PR DESCRIPTION
**Pull request by ahilali**
This module was written to extend the functionality of Product Variants
and allow you to enable or disable product variants with two buttons

### CODE REVIEW

File: product_variant_inactive/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ :alt: License: AGPL-3
+
+Module name
```
Comment (elicoidal) [inline product_variant_inactive/README.rst]: Hi @ahilali thanks for your contribution
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
Add line before as this is the document title

File: product_variant_inactive/README.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+===========
+
+This module was written to extend the functionality of Product Variants
```
Comment (elicoidal) [inline product_variant_inactive/README.rst]: This module extends the functionality of product variants
adding 2 buttons in tree view to easily activate and deactivate them.

File: product_variant_inactive/README.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+
+Installation
+============
+
+To install this module, you need to:
+
+*To install Product module
+
+For further information, please visit:
```
Comment (elicoidal) [inline product_variant_inactive/README.rst]: remove forum section

File: product_variant_inactive/README.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+* Sebastien BEAU <sebastien.beau@akretion.com>
+* Abdessamad HILALI <abdessamad.hilali@akretion.com>
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+ :alt: Odoo Community Association
+ :target: http://odoo-community.org
```
Comment (elicoidal) [inline product_variant_inactive/README.rst]: https (x2)

File: product_variant_inactive/__init__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline product_variant_inactive/__init__.py]: Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: product_variant_inactive/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+{
```
Comment (elicoidal) [inline product_variant_inactive/__openerp__.py:5]: Please follow 
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py
version: 8.0.1.0.0

File: product_variant_inactive/README.rst [hunk 7]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ :alt: License: AGPL-3
+
+Module name
+===========
+
+This module was written to extend the functionality of Product Variants
+and allow you to make activate or deactivate products using the two buttons
+
+.. image:: img/img_01.png
```
Comment (elicoidal) [inline product_variant_inactive/README.rst]: img.png seems not referenced here (I would remove it from the module as the other one is explicit)

Comment (elicoidal) [inline product_variant_inactive/README.rst]: For screenshot please trim to the necessary information (no need browser title or your desktop)

**Comment by bealdav** [comment]
ping @ahilali

**Comment by bealdav** [comment]
READY TO MERGE

Thanks for a review

**Comment by mourad-ehm** [comment]
Hi,
I tested it. it works fine.
:+1:

**Comment by sebastienbeau** [comment]
:+1: tested

**Pull request merged**